src/endpoints/__tests__/createCitiesEndpoint.spec.js

```diff
@@ -14,7 +14,10 @@ describe('cities', () => {
     events: true,
     extras: true,
     name_without_prefix: 'Augsburg',
-    prefix: null
+    prefix: null,
+    longitude: 10.89779,
+    latitude: 48.3705449,
+    aliases: { 'augsburg': { long: 10.89779, lat: 48.3705449 } }
```

Comment:
'augsburg' should be captalized. Also the string 'Gersthofen' would make more sense here.

Reply:
'long' and 'lat' has been changed to 'longitude' and 'latitude' in the API.